src/results/proforma.jl

```diff
@@ -359,6 +364,72 @@ function update_metrics(m::Metrics, p::REoptInputs, tech::AbstractTech, tech_nam
     nothing
 end
 
+function update_ghp_metrics(m::REopt.Metrics, p::REoptInputs, tech::REopt.AbstractTech, tech_name::String, results::Dict, third_party::Bool)
```

Comment:
@rathod-b I updated this to handle Central GHP "WWHP" with the two different heat pump sizes. I just made `new_kw` == average of two sizes as a workaround even though they should be handled separately. Punt on that!

Reply:
Makes sense!